lib/membrane/video_compositor/request.ex

```diff
@@ -22,16 +22,20 @@ defmodule Membrane.LiveCompositor.Request do
 
   @spec register_input_stream(
           LiveCompositor.input_id(),
-          :inet.port_number(),
+          :inet.port_number() | LiveCompositor.port_range(),
           :inet.port_number()
         ) ::
           request_result()
-  def register_input_stream(input_id, input_port_number, lc_port) do
+  def register_input_stream(input_id, port, lc_port) do
     %{
       type: :register,
-      entity_type: :input_stream,
+      entity_type: :rtp_input_stream,
       input_id: "#{input_id}",
-      port: input_port_number,
+      port:
+        case port do
+          {start, endd} -> "#{start}:#{endd}"
+          exact_port -> exact_port
+        end,
```

Comment:
I think this looks better than putting the case statement inside the map declaration
```elixir
port = case port do ...
```

```elixir
%{
  port: port
}
```